### DISCUSSION TITLE
PROXIES

### DISCUSSION THREAD

**Discussion by pl4udible** [3kh0/ext-remover#523] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
PLEASE CAN I HAVE PROXXIES THAT ARE WORKING 

ESPECIALLY HERUKO APP

ANY ONES THAT WORK AND ARE NOT BLOCKED

**Comment by qDread-dev** [comment, marked as answer]
Most servers that have proxies like TitaniumNetwork / Nebula have X amout of free proxies, just use those

**Comment by obkduishinijf** [comment]
> PLEASE CAN I HAVE PROXXIES THAT ARE WORKING
> 
> ESPECIALLY HERUKO APP
> 
> ANY ONES THAT WORK AND ARE NOT BLOCKED

1. stop yelling
2. if glitch.me or .io is blocked, then i can't help you

**Reply by qDread-dev** [reply]
Its always great when kids make a github account just to complain they dont have a proxy lmao

**Reply by obkduishinijf** [reply]
> Its always great when kids make a github account just to complain they dont have a proxy lmao

i find it ironic, because it's super simple to find them

**Reply by qDread-dev** [reply]
> > Its always great when kids make a github account just to complain they dont have a proxy lmao
> 
> i find it ironic, because it's super simple to find them

For real, for them to even know about proxies, they are most likely in a discord that has a dispenser bot.

**Reply by obkduishinijf** [reply]
> > > Its always great when kids make a github account just to complain they dont have a proxy lmao
> > 
> > 
> > i find it ironic, because it's super simple to find them
> 
> For real, for them to even know about proxies, they are most likely in a discord that has a dispenser bot.

yeah, i only discovered proxies because i was tired of restrictions, i'm still trying to figure out a way to turn off goguardian without point blank, as that causes lag. might just downgrade 🤷

**Reply by qDread-dev** [reply]
> > > > Its always great when kids make a github account just to complain they dont have a proxy lmao
> > > 
> > > 
> > > i find it ironic, because it's super simple to find them
> > 
> > 
> > For real, for them to even know about proxies, they are most likely in a discord that has a dispenser bot.
> 
> yeah, i only discovered proxies because i was tired of restrictions, i'm still trying to figure out a way to turn off goguardian without point blank, as that causes lag. might just downgrade 🤷

There is the new point blank, which I think works for go guardian

**Reply by obkduishinijf** [reply]
> > > > > Its always great when kids make a github account just to complain they dont have a proxy lmao
> > > > 
> > > > 
> > > > i find it ironic, because it's super simple to find them
> > > 
> > > 
> > > For real, for them to even know about proxies, they are most likely in a discord that has a dispenser bot.
> > 
> > 
> > yeah, i only discovered proxies because i was tired of restrictions, i'm still trying to figure out a way to turn off goguardian without point blank, as that causes lag. might just downgrade 🤷
> 
> There is the new point blank, which I think works for go guardian

yeah, point blank works for all of my extensions, it just lags the chromebook.

**Reply by Aka-but-nice** [reply]
This is going to far

**Reply by Aka-but-nice** [reply]
> > > > Its always great when kids make a github account just to complain they dont have a proxy lmao
> > > 
> > > 
> > > i find it ironic, because it's super simple to find them
> > 
> > 
> > For real, for them to even know about proxies, they are most likely in a discord that has a dispenser bot.
> 
> yeah, i only discovered proxies because i was tired of restrictions, i'm still trying to figure out a way to turn off goguardian without point blank, as that causes lag. might just downgrade 🤷

Also, if you downgrade, which exploit are you considering, swamp wouldn't work because that's the extensions update,
not the device update.
I recommend you use lacros96, which is in TN Kajigs

**Comment by bartholemuew** [comment]
goo gle
